src/components/cardheader/CardHeader.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardHeader = props => {
+  const { title, icon = null } = props;
+
+  return (
+    <div className="card-header">
+      <div className="card-header__l">
+        <div className="title">{title}</div>
+      </div>
+      {!icon ? (
```

Comment:
you can use another form of conditional rendering `icon && <div />`
For more info:
https://reactjs.org/docs/conditional-rendering.html

Reply:
[form-preview c718227] form-preview:#2 - add conditional rendering

Reply:
[form-preview 3cc943e] form-preview:#2 - added new line and conditional rendering